### PR TITLE
Report invalid or missing packit config

### PR DESCRIPTION
**Pull request by dhodovsk**
Requires https://github.com/packit-service/packit/pull/664
Fixes https://github.com/packit-service/packit-service/issues/222

### CODE REVIEW

File: packit_service/worker/github_handlers.py [hunk 1]
```diff
@@ -78,6 +78,27 @@ class AbstractGithubJobHandler(JobHandler):
...
+        if not package_config:
+            raise PackitConfigException(
+                f"No config file found in {project.full_repo_name}"
+            )
+    except PackitConfigException as ex:
+        if pr_id:
+            project.pr_comment(pr_id, ex)
+        else:
+            issues = project.get_issue_list()
+            if "Invalid packit config" not in [x.title for x in issues]:
```
Comment (TomasTomecek) [inline packit_service/worker/github_handlers.py]: in future, this should be stored in the database

File: packit_service/worker/github_handlers.py [hunk 2]
```diff
@@ -78,6 +78,27 @@ class AbstractGithubJobHandler(JobHandler):
...
+        package_config: PackageConfig = get_package_config_from_repo(project, reference)
+        if not package_config:
+            raise PackitConfigException(
+                f"No config file found in {project.full_repo_name}"
+            )
+    except PackitConfigException as ex:
+        if pr_id:
+            project.pr_comment(pr_id, ex)
+        else:
+            issues = project.get_issue_list()
```
Comment (TomasTomecek) [inline packit_service/worker/github_handlers.py]: this will be pretty beefy on projects with thousands of issues

Comment (lachmanfrantisek) [reply packit_service/worker/github_handlers.py]: If GitHub supports filtering, this will be a nice improvement to OGR..;)

Comment (lachmanfrantisek) [reply packit_service/worker/github_handlers.py]: https://github.com/packit-service/ogr/issues/308

File: packit_service/worker/github_handlers.py [hunk 3]
```diff
@@ -75,6 +75,29 @@ class AbstractGithubJobHandler(JobHandler):
...
+            )
+    except PackitConfigException as ex:
+        if pr_id:
+            project.pr_comment(pr_id, ex)
+        else:
+            # TODO: filter when https://github.com/packit-service/ogr/issues/308 fixed
+            issues = project.get_issue_list()
+            if "Invalid packit config" not in [x.title for x in issues]:
+                # TODO: store in DB
+                i = project.create_issue("[packit] Invalid config", ex)
```
Comment (lachmanfrantisek) [inline packit_service/worker/github_handlers.py]: We can enhance the format later, but some ideas:

- formating -- is it "safe" to print the exception directly? (Maybe yes.) Or use a ``` block.
- links to the documentation
- contact with our team
- info about the issue workflow (e.g. "We will try not to do anything until you close the issue...")

**Pull request merged**